backend/app.py

```diff
@@ -449,12 +449,55 @@ def new_hierarchy_stuff(
     # return json.loads(df.to_json(orient='records'))
 
 
-@APP.get("/update-cset")  # maybe junk, or maybe start of a refactor of above
-def update_cset(
-    codeset_id: int, concept_id: int, state: bool
-) -> Dict:
-    modification = {'codeset_id': codeset_id, 'concept_id': concept_id, 'state': state}
-    print(f'update-cset: {codeset_id}, {concept_id}, {state}')
+# TODO
+#  - example case: http://localhost:8000/update-cset-concept-inclusion?codeset_id=496860542&concept_ids=35787839&concept_ids=35787840
+#  - old example case pre param change: http://localhost:8000/update-cset-concept-inclusion?codeset_id=496860542&concept_ids=35787839&state=false
+# TODO: frontend change
+#  We should change this to `concept_ids`: When a box is checked/unchecked, send frontend should send *all* the concept_ids to be included
+@APP.get("/update-cset-concept-inclusion")
+def update_cset(codeset_id: int, concept_ids: Union[List[int], None] = Query(default=None)) -> Dict:
```

Comment:
@Sigfried Above and below this function are a bunch of comments with some TODOs writing out what I need to do to complete this issue.